src/babel-plugin.js

```diff
@@ -39,6 +39,7 @@ export default ({ types: t }) => {
           // Push strings template literals to gql calls
           // e.g. gql(`query myQuery {}`) -> query myQuery {}
           if (
+            this.gqlIdentifierName &&
```

Comment:
I would still wanna know why it fails, because it shouldn't parse non `gql` function calls. Your solution just avoids it, but doesn't fix it.